### ISSUE TITLE
Out of memory after loading ARITH.MUS

### ISSUE THREAD

**Issue by alexshendi** [jlfaucher/FormMath#1]
I can create MUSIMP.SYS by executing:
```
(load "musimp.lsp")
reclaim();
save("musimp.sys");
```
But after executing:
```
rds("arith.mus");
```

I get an out of memory error both under dosbox and DOS 5.0.
I'm using MULISP-87 and the MUSIMP.LSP from this repository.

Any ideas what I could do?

**Comment by fgrub** [comment]
Hi,
your FormMath is great, but there's a bug with some expressions.

Examples (after loading "mulisp mumathfm.sys"):
2^(1/2); is ok
(1/2)^(1/2);  produces an error!
This happens for every expression a^(1/2) where a is of the form (x/y).

The strange thing is, that if you press C (Continue) after the error, you'll nevertheless get the correct result.
I hope you can fix this bug, because this 2D output of your muMathFM is very nice.

**Comment by jlfaucher** [comment]
Indeed, I can reproduce the problem but have no time to investigate.
It appears that I use modified MuMath 83 files.
I can't publish the modified sources because they are based on files that are copyrighted by The Soft Warehouse.
As a workaround, I provide a pre-built MUMATHFM.SYS in the BUILD directory.
